hikari/audit_logs.py

```diff
@@ -260,7 +364,41 @@ class MemberMoveEntryInfo(MemberDisconnectEntryInfo):
     """Extra information for the voice chat based member move entry."""
 
     channel_id: snowflakes.Snowflake = attr.ib(repr=True)
-    """The amount of members who were disconnected from voice in this entry."""
+    """The channel that the member(s) have been moved to"""
+
+    async def fetch_channel(self) -> channels.GuildVoiceChannel:
+        """Fetch the guild voice based channel where the member(s) have been moved to.
+
+        Returns
+        -------
+        hikari.channels.GuildVoiceChannel
+            The guild voice based channel where the member(s) have been moved to.
```

Comment:
This will only ever be 1 member, not multiple

Reply:
are you sure? because the docstring for the channel id says member(s) and also count is the amount of members
![image](https://user-images.githubusercontent.com/44059343/96681722-68355780-1380-11eb-94d1-524f008ae6c5.png)